src/main/resources/localization/defaultMessages.po

```diff
@@ -0,0 +1,17 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2025 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# Ioan Faur <fioan89@gmail.com>, 2025.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Coder Toolbox 1.0\n"
+"Report-Msgid-Bugs-To: support@coder.com\n"
```

Comment:
```suggestion
"Report-Msgid-Bugs-To: jetbrains-plugins@coder.com\n"
```
We want to keep it separate from support.

Reply:
Done. Good to know.